deploy/kubernetes/docker/rss_base_centos.Dockerfile

```diff
@@ -0,0 +1,24 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# this docker file serves the rss base image for CentOS/RedHat distributions
+ARG BASE_IMAGE=eclipse-temurin:8-jdk-centos7
+FROM ${BASE_IMAGE}
+
+RUN yum install lzo zlib zlib-devel lzop lsof netcat dnsutils less procps iputils-ping -y && \
+    yum clean all && \
+    rm -rf /var/cache/yum
```

Comment:
add a new line

Reply:
Is `rss_base_centos.Dockerfile` a good name?
Maybe `base/centos/Dockerfile` will be better.